README.md

```diff
@@ -35,10 +35,16 @@ The project is open to contributions. Feel free to open an issue or even a pull-
 
 1. Clone the repository (or fork it if you intend to contribute)
 2. `npm install`
-3. Start a PostgreSQL server. You can run `./start-local-db.sh` if you don’t have a server already.
+3. Start a PostgreSQL server. You can run `./scripts/start-local-db.sh` if you don’t have a server already.
 4. Copy the file `.env.example` as `.env`
 5. `npm run dev`
 
+## Run in a container
+1. Run `sudo npm run build-image` to build the docker image from the Dockerfile
+2. Copy the file `container.env.example` as `container.env`
+3. Run `sudo npm run start-container` to start the postgres and the spliit2 containers
```

Comment:
```suggestion
1. Run `npm run build-image` to build the docker image from the Dockerfile
2. Copy the file `container.env.example` as `container.env`
3. Run `npm run start-container` to start the postgres and the spliit2 containers
```

I’m not sure that `sudo` is required, I was able to run the container version without it. Anything I’m missing?

Reply:
I'm not sure why but I needed it when running under npm but, it makes sense not to have that in the README.